codespell_lib/data/dictionary.txt

```diff
@@ -32388,8 +32438,12 @@ segmetned->segmented
 segmetns->segments
 segument->segment
 seguoys->segues
+segway->segue
```

Comment:
Could be a [Segway](https://en.wikipedia.org/wiki/Segway) PT. Move to _rare_?

Reply:
I don't know. It's technically just a brand name and not a proper word, just like the new hip start-up `Frobnicatr` is just a typo for `Frobnicator` and should probably be treated as such. I'd ask projects referring to such brands to fix it locally, in their project ignore file or similar.

WDYT?

Reply:
Agreed, let's avoid it. Besides, that brand is less fashionable nowadays.